src/Configurations/test/Responses.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { waitFor, screen } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import * as components from '@folio/stripes-acq-components';
+
+import { mockKy, server } from '../../test/net';
+import {
+  mockedConfigurations, mockedProviders,
+  mockedSingleConfiguration,
+  renderConfigurations,
+} from './setup';
+
+
+jest.mock('react-virtualized-auto-sizer', () => ({ children }) => children({ width: 1920, height: 1080 }));
+
+jest.mock('@folio/stripes-components/lib/Icon', () => props => (
+  <span data-testid={props['data-testid']}>
+    <svg />
+    {props.children && <span>{props.children}</span>}
+  </span>
+));
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  useShowCallout: jest.fn(),
+}));
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useOkapiKy: () => mockKy,
+  useStripes: () => ({
+    hasPerm: jest.fn().mockReturnValue(true),
+  }),
+  IfPermission: props => <>{props.children}</>,
+}));
+
+
+const renderSingleConfiguration = async () => {
+  renderConfigurations();
+
+  const cell = await screen.findByRole('gridcell', { name: 'RS1' });
+  user.click(cell);
+};
+
+const renderConfigurationEdit = () => {
+  renderConfigurations('/1/edit');
+};
+
+
+const mockShowCallout = jest.fn();
+
+beforeAll(async () => {
+  jest.spyOn(components, 'useShowCallout').mockImplementation(() => mockShowCallout);
+});
+
+afterEach(() => {
+  mockShowCallout.mockClear();
+});
+
+describe('Fetching single configuration', async () => {
+  it('Does not show error callout in EditorLayer, if there are not errors', async () => {
+    server.use(
+      mockedConfigurations(),
+      mockedSingleConfiguration(),
+    );
+
+    await renderConfigurationEdit();
+
+    await waitFor(() => expect(mockShowCallout).not.toBeCalled());
```

Comment:
This assertion won't break if we mock an error here.
That's because the waiting for `mockShowCallout` not to be called is fulfilled immediately: it's not called from the start.